tests/helpers.py

```diff
@@ -0,0 +1,52 @@
+"""Module to help to create tests."""
+from unittest.mock import Mock
+
+from kytos.core import Controller
+from kytos.core.config import KytosConfig
+
+
+def get_controller_mock():
+    """Return a controller mock."""
+    options = KytosConfig().options["daemon"]
+    controller = Controller(options)
+    controller.log = Mock()
+    return controller
+
+
+def get_mocked_requests(_):
+    """Mock requests.get."""
+    return MockResponse(
+        {
+        },
+        200,
+    )
+
+
+class MockResponse:
+    """
+    Mock a requests response object.
+
+    Just define a function and add the patch decorator to the test.
+    Example:
+    def mocked_requests_get(*args, **kwargs):
+        return MockResponse({}, 200)
+    @patch('requests.get', side_effect=mocked_requests_get)
+
+    """
+
+    def __init__(self, json_data, status_code):
+        """Create mock response object with parameters.
+
+        Args:
+            json_data: JSON response content
+            status_code: HTTP status code.
+        """
+        self.json_data = json_data
+        self.status_code = status_code
+
+    def json(self):
+        """Return the response json data."""
+        return self.json_data
+
+    def __str__(self):
+        return self.__class__.__name__
```

Comment:
This code is repeated over and over on all NApps. I think we should consider using an external library for this, like https://github.com/getsentry/responses or https://requests-mock.readthedocs.io/en/latest/

Reply:
Great idea @ajoaoff, `reponses` is very convenient to use. @rmotitsuki  could you map this for us in the issues to be addressed in the future? It seems like this mocked response also isn't being used at the moment, so maybe it could be delivered with `responses` in the future? What do you guys think?